docs/pages.rst

```diff
@@ -29,3 +29,36 @@ Files
 Additional files, such as images, can be uploaded so they can be referenced in page.
 These files are placed in MEDIA_ROOT/pages_files by default. This location needs to
 be writable by the webserver for uploads to work.
+
+Maintaining pages in files
+==========================
+
+Pages live in the database, and can be edited through the web UI.
+However, it can be useful to store them in files (e.g. in a git repo,
+with the site source code).
+There is a management command (``load_pages``) that will read pages from
+files into the database.
+
+Pages must be stored as markdown in a directory, in the same hierarchy
+as the desired URL structure.
+The ``PAGE_DIR`` setting should be an absolute path to root directory of this
+hierarchy, beginning and ending with ``/``.
+e.g.::
+
+    /app/pages/                         ← PAGE_DIR
+    /app/pages/index.md                 ← Home Page: /
+    /app/pages/about.md                 ← Container Page: /about/
```

Comment:
Yes, it does. But that doesn't match how we actually render pages. The root index page is a hack. This closely mirrors the DB structure.